src/server/db/models/github-user.js

```diff
@@ -7,9 +7,32 @@ export default function register(db) {
    *   created_at: creation date
    *   updated_at: update date
    *   last_login_at: last login date
+   *   snaps_added: number of snaps added
+   *   snaps_removed: number of snaps removed
+   *   names_registered: number of snap names registered
```

Comment:
Yes.  To quote Evan, "we can add" [or presumably actually subtract] "them together at the level of Grafana".

For the moment the metric only cares about when this is non-zero; the idea is to count developers who've got past certain steps in the process.